extra_data/export.py

```diff
@@ -12,215 +12,32 @@
 
 from argparse import ArgumentParser
 import os.path as osp
-from queue import Queue
-from threading import Event, Thread
-from time import time
 from warnings import warn
 
-import msgpack
-import numpy as np
-import zmq
```

Comment:
I guess we can also drop the direct requirements on msgpack & pyzmq in `setup.py`, though they'll still be installed as dependencies of karabo bridge.

Reply:
Right, thanks